### DISCUSSION TITLE
Request for Stretch layout option

### DISCUSSION THREAD

**Discussion by KnivezOfficial** [rafaelvcaetano/melonDS-android#1295] [Ideas]
Hello good day, I just want to request an option to stretch screen layout, I already know that there is an option to resize it but if you are emulating on a modern phone it still leaves a black bar on the sides or any part of the screen. stretching to fill up those remaining spaces when screen is side by side or any orientation will be helpful, just like the option on drastic ds, Hope this will be implemented in the future. Thank you and will be supporting this project ❤️

**Comment by Ara1705** [comment]
Would also make melonDS able to support widescreen patches, making it the de facto best android NDS emulator (support for WFC, plus aspect ratio mods, and retroachievements support)
